src/assets/stylesheets/Tabs.scss

```diff
@@ -2,6 +2,20 @@
 @use "./rpf_design_system/font-weight" as *;
 @use "./rpf_design_system/colours" as *;
 
+:root {
+  .--light {
+    --rpf-tab-button-hover: var(--rpf-grey-100);
+    --rpf-tab-button-background: inherit;
+    --rpf-tab-border-bottom-selected: var(--rpf-teal-900);
+  }
+
+  .--dark {
+    --rpf-tab-button-hover: var(--rpf-grey-900);
+    --rpf-tab-button-background: var(--rpf-grey-800);
+    --rpf-tab-border-bottom-selected: var(--rpf-teal-800);
+  }
+}
```

Comment:
Looks like these aren't being applied on the web component test page 🤔

Reply:
Tidy-up ticket